pkg/sqlite/configmap.go

```diff
@@ -100,76 +104,84 @@ func (c *ConfigMapLoader) Populate() error {
 	c.log.Info("loading Bundles")
 	csvListYaml, ok := c.configMapData[ConfigMapCSVName]
 	if !ok {
-		return fmt.Errorf("couldn't find expected key %s in configmap", ConfigMapCSVName)
+		errs = append(errs, fmt.Errorf("couldn't find expected key %s in configmap", ConfigMapCSVName))
+		return utilerrors.NewAggregate(errs)
 	}
 	csvListJson, err := yaml.YAMLToJSON([]byte(csvListYaml))
 	if err != nil {
-		c.log.WithError(err).Debug("error loading CSV list")
-		return err
+		errs = append(errs, fmt.Errorf("error loading CSV list: %s", err))
+		return utilerrors.NewAggregate(errs)
 	}
 
 	var parsedCSVList []v1alpha1.ClusterServiceVersion
 	err = json.Unmarshal([]byte(csvListJson), &parsedCSVList)
 	if err != nil {
-		c.log.WithError(err).Debug("error parsing CSV list")
-		return err
+		errs = append(errs, fmt.Errorf("error parsing CSV list: %s", err))
```

Comment:
This is fine given the goals of the PR are really geared towards appregistry, but -

if we wanted to be permissive here, wouldn't we treat this as a generic list, and attempt to parse each CSV one by one? As it stands now, if one doesn't parse, the whole thing will bail.